### DISCUSSION TITLE
[feature request] 增加2倍数或以上的播放速度

### DISCUSSION THREAD

**Discussion by TakChen** [xfangfang/wiliwili#293] [Ideas]
可不可能增加3倍数或4倍数等选项? 
或者 新增两个可自定义的栏位, 让用户自己填? 
> 因为有时候我自己在网页版的时候会 用3倍数或4倍数

**Comment by xfangfang** [comment]
Done by: https://github.com/xfangfang/wiliwili/actions/runs/8066525705

**Reply by TakChen** [reply]
感谢!!! 知道地方了, 后面我可以自己改了😘
